guides/hack/03-expressions-and-operators/91-coalesce-examples/assignment.hack

```diff
@@ -0,0 +1,38 @@
+use namespace HH\Lib\{Dict, Str};
```

Comment:
It'd be nice to declare a namespace here; most examples do, and it avoids polutting the root namespace which can get annoying in the actual site code